### PR TITLE
Sample added for Customize Word to HTML

### PR DESCRIPTION
**Pull request by Venkateshmuruganandam**
Hi All,
Sample added for Customize Word to HTML

Regards
Venkatesh

### CODE REVIEW

File: HTML-conversions/Customize-Word-to-HTML-conversion/.NET/Customize-Word-to-HTML-conversion/Program.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                    //Exports the text form fields as editable 
+                    document.SaveOptions.HtmlExportTextInputFormFieldAsText = false;
+                    //Sets the style sheet type
+                    document.SaveOptions.HtmlExportCssStyleSheetType = CssStyleSheetType.Inline;
+                    //Set value to omit XML declaration in the exported html file.
+                    //True- to omit xml declaration, otherwise false.
+                    document.SaveOptions.HtmlExportOmitXmlDeclaration = false;
+
+                    //Creates file stream.
+                    using (FileStream outputFileStream = new FileStream(Path.GetFullPath(@"../../../RtfToHTML.html"), FileMode.Create, FileAccess.ReadWrite))
```
Comment (MohanaselvamJothi) [inline HTML-conversions/Customize-Word-to-HTML-conversion/.NET/Customize-Word-to-HTML-conversion/Program.cs:31]: Please change as Word to HTML.html file name

File: HTML-conversions/Customize-Word-to-HTML-conversion/.NET/Customize-Word-to-HTML-conversion/Program.cs [hunk 2]
```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using Syncfusion.DocIO;
+using Syncfusion.DocIO.DLS;
+using System.IO;
+using static System.Collections.Specialized.BitVector32;
```
Comment (MohanaselvamJothi) [inline HTML-conversions/Customize-Word-to-HTML-conversion/.NET/Customize-Word-to-HTML-conversion/Program.cs:5]: Check and remove unwanted namespaces

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: HTML-conversions/Customize-Word-to-HTML-conversion/.NET/Customize-Word-to-HTML-conversion/Program.cs (hunk 1 above)
Comment (Venkateshmuruganandam) [reply HTML-conversions/Customize-Word-to-HTML-conversion/.NET/Customize-Word-to-HTML-conversion/Program.cs]: updated

File: HTML-conversions/Customize-Word-to-HTML-conversion/.NET/Customize-Word-to-HTML-conversion/Program.cs (hunk 2 above)
Comment (Venkateshmuruganandam) [reply HTML-conversions/Customize-Word-to-HTML-conversion/.NET/Customize-Word-to-HTML-conversion/Program.cs]: updated

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**